src/core/freebsd/SDL_evdev_kbd_freebsd.c

```diff
@@ -87,15 +87,17 @@ static int fatal_signals[] =
 
 static void kbd_cleanup(void)
 {
+    struct mouse_info mData;
     SDL_EVDEV_keyboard_state* kbd = kbd_cleanup_state;
     if (kbd == NULL) {
         return;
     }
     kbd_cleanup_state = NULL;
-    
+    mData.operation = MOUSE_SHOW;
```

Comment:
Does `mData` need to be initialized here? It was initialized explicitly in `SDL_EVDEV_kbd_init`.

Reply:
It has to be intialized here.